src/Core/functions.php

```diff
@@ -153,13 +153,9 @@ function pr($var)
      */
     function pj($var)
     {
-        if (!Configure::read('debug')) {
```

Comment:
OK.
I'm not sure what caused this to break, should i make another pull request or just change in here?